gradle.properties

```diff
@@ -77,3 +77,4 @@ protobuf.version=3.17.1
 scott-data-hsqldb.version=0.1
 servlet.version=4.0.1
 slf4j.version=1.7.25
+log4j2.version=2.15.0
```

Comment:
can you move this into alphabetical order

Reply:
fixing this and committing now.